libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -1237,6 +1250,14 @@ class MapboxNavigation @VisibleForTesting internal constructor(
         }
     }
 
+    private fun <T> runInTelemetryContext(func: (MapboxNavigationTelemetry) -> T): T? {
```

Comment:
I guess it let the compiler inline lambdas
```suggestion
    private inline fun <T> runInTelemetryContext(func: (MapboxNavigationTelemetry) -> T): T? {
```